src/service/worryService.ts

```diff
@@ -0,0 +1,25 @@
+import { ClientException } from "../common/error/exceptions/customExceptions";
+import statusCode from "../constants/statusCode";
+import { worryRepository, categoryRepository } from "../repository"
+
+const getWorryListByCategoryId = async (categoryId: number) => {
+    const worryWithList = (categoryId == 0) ? await worryRepository.getWorries() : await worryRepository.getWorryListByCategoryId(categoryId);
```

Comment:
p3. 전 이렇게 해도 상관 없을 것 같아요! 근데 조건문의 의미가 모호하니 저라면 함수로 빼주겠어요 ㅎㅎ
```suggestion
 const isTotal(category: number): boolean => categoryId === 0;
 
 const getWorryListByCategoryId = async (categoryId: number) => {
    const worryWithList = isTotal(categoryId) ? await worryRepository.getWorries() : await 
    worryRepository.getWorryListByCategoryId(categoryId);
    // ...
 }
```

Reply:
훨씬 이해하기 좋은 코드인것 같습니다..!!!!!! 감사합니다~!~!~!